docs/uz/docs/index.md

```diff
@@ -0,0 +1,45 @@
+<p align="center">
+  <a href="https://fastapi.tiangolo.com"><img src="https://fastapi.tiangolo.com/img/logo-margin/logo-teal.png" alt="FastAPI"></a>
+</p>
+<p align="center">
+    <em>FastAPI yuqori darajali, o'rganishga oson, tez ishlab chiqish imkonini beruvchi framework</em>
+</p>
+<p align="center">
+<a href="https://github.com/tiangolo/fastapi/actions?query=workflow%3ATest+event%3Apush+branch%3Amaster" target="_blank">
+    <img src="https://github.com/tiangolo/fastapi/workflows/Test/badge.svg?event=push&branch=master" alt="Test">
+</a>
+<a href="https://coverage-badge.samuelcolvin.workers.dev/redirect/tiangolo/fastapi" target="_blank">
+    <img src="https://coverage-badge.samuelcolvin.workers.dev/tiangolo/fastapi.svg" alt="Coverage">
+</a>
+<a href="https://pypi.org/project/fastapi" target="_blank">
+    <img src="https://img.shields.io/pypi/v/fastapi?color=%2334D058&label=pypi%20package" alt="Package version">
+</a>
+<a href="https://pypi.org/project/fastapi" target="_blank">
+    <img src="https://img.shields.io/pypi/pyversions/fastapi.svg?color=%2334D058" alt="Supported Python versions">
+</a>
+</p>
+
+---
+
+**Dokumentatsiya**: <a href="https://fastapi.tiangolo.com" target="_blank">https://fastapi.tiangolo.com</a>
+
+**Manba kodi**: <a href="https://github.com/tiangolo/fastapi" target="_blank">https://github.com/tiangolo/fastapi</a>
+
+---
+
+FastAPI standart Python type hintlariga asoslangan holda Python 3.7+ versiyasi bilan APIlar qurish uchun zamonaviy, tez (yuqori darajali) web framework.
+
+Asosiy hususiyatlari:
+
+* **Tez**: Juda yuqori darajali, **NodeJS** va **Go** bilan teng (Starlette va Pydantic ga rahmat). [Mavjud bo'lgan eng tez python framworklari](#performance).
+* **Tez kodlash**: Xususiyatlarni ishlab chiqish tezligi 200% dan 300% gacha oshishi. *
+* **Xatolar kamligi**: Insonlar (dasturchilar) tomonidan keltirib chiqarilgan xatolarning 40% ni kamaytiring. *
+* **Intuitive**: Great editor support. <abbr title="also known as auto-complete, autocompletion, IntelliSense">Completion</abbr> everywhere. Less time debugging.
+* **Oson**: Oson o'rganish va foydalanish uchun mo'ljallangan. Dokumentatisyani o'qish uchun kamroq vaqt.
+* **Qisqa**: Kod dublikatlarni minimallashtir. Xar bir parametr deklaratsiyasidan bir nechta xususiyatlar. Kamroq xatolar.
+* **Mustahkam**: Production uchun tayyor kodni oling. Avtomatik interaktiv hujjatlar bilan.
+* **Standartlarga asoslangan**: API lar uchun ochiq standartlarga asoslangan (va to'liq moslashtirilgan): <a href="https://github.com/OAI/OpenAPI-Specification" class="external-link" target="_blank">OpenAPI</a> (oldinroq Swagger deb tanilgan) va <a href="https://json-schema.org/" class="external-link" target="_blank">JSON Schema</a>.
+
+<small>* ichki ishlab chiquvchi va production ilovalarni qurayotgan  jamoaning testlariga asoslangan baholash</small>
+
+## Sponsors
```

Comment:
It looks like this page hasn't been translated all the way through. Specifically, this page now starts with a section "Sponsors" which doesn't have any content.

Do you have time to continue the translation for this page in full?